src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,11 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Returns true if command is mutating
+     */
+    public boolean isMutating(){
```

Comment:
Good job updating the header comment. It would be better to mention that it returns true if the command *potentially* mutates the data. Try to be as precise as possible with header comments.